package.json

```diff
@@ -35,6 +35,22 @@
     "watch:peregrine": "cd packages/peregrine && npm run -s watch; cd - >/dev/null",
     "watch:venia": "cd packages/venia-concept && npm run -s watch; cd - >/dev/null"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && npm test"
```

Comment:
We talked about it offline and we'd prefer a `pre-push` hook that runs `prettier` and `lint` instead of a `pre-commit` hook.

Reply:
I like running tests in there too 👌 , but yes, pre-push please.